src/main/webapp/scripts/calendar-script.js

```diff
@@ -0,0 +1,190 @@
+import {initPromise} from './account-info.js';
+
+const EVENTS_CALENDAR = 'c_96sboq7a00dhtc9c24pr7mpi3o@group'+
+    '.calendar.google.com';
+
+initPromise.then(function() {
+  gapi.client.load('calendar', 'v3', function() {
+    getCalendarEvents(EVENTS_CALENDAR, false).then(function(events) {
+      if (events.length > 0) {
+        for (let i = 0; i < events.length; i++) {
+          const event = events[i];
+          addEventToPage(event);
+        }
+      }
+    });
+  });
+});
+
+
+/**
+ * Gets events from a calendar with specified ID
+ * @param {String} id ID of calendar we want events from
+ * @param {Boolean} deleted Whether or not we want to include deleted events
+ * @return {Promise} Promise which resolves after the
+ *    events of the calendar are listed
+ */
+function getCalendarEvents(id, deleted) {
+  return gapi.client.calendar.events.list({
+    'calendarId': id,
+    'timeMin': (new Date()).toISOString(),
+    'showDeleted': deleted,
+    'singleEvents': true,
+    'orderBy': 'startTime',
+  }).then(function(response) {
+    const events = response.result.items;
+    return events;
+  });
+}
+
+/**
+ * Adds an event to the webpage
+ * @param {Event} event Event to add
+ */
+function addEventToPage(event) {
+  const summary = event.summary;
+  const startDateTime = event.start.dateTime;
+  const endDateTime = event.end.dateTime;
+  const description = event.description;
+
+  const listElement = document.createElement('LI');
+  const leftDiv = document.createElement('div');
+
+  // summary
+  const heading = document.createElement('h3');
+  heading.appendChild(document.createTextNode(summary));
+  leftDiv.appendChild(heading);
+
+  // Time
+  const monthNames = ['January', 'February', 'March', 'April', 'May',
+    'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+  const timeNode = document.createElement('h4');
+  const start = new Date(startDateTime);
+  const end = new Date(endDateTime);
+  const startDate = monthNames[start.getMonth()] + ' ' +
```

Comment:
I'm pretty sure there are time formatting utilities for these, but it's ok for now for the mad dash :)